bootstrap3-typeahead.js

```diff
@@ -223,17 +224,17 @@
             }
 
             this.shown = true;
-            this.$element.attr('aria-expanded', 'true');
-            this.$element.attr('aria-owns', getOrGenId(element[0]));
+            this.$parent.attr('aria-expanded', 'true');
+            this.$parent.attr('aria-owns', getOrGenId(element[0]));
```

Comment:
It's correct for `aria-owns` to move to the container but `aria-controls` should be defined on the text field, pointing at the same thing